src/owncloud/apache_config

```diff
@@ -11,6 +11,20 @@ Alias "/apps" "${SNAP_DATA}/owncloud/apps"
     Require all granted
 </Directory>
 
-<IfModule dir_module>
-    DirectoryIndex index.html index.php
-</IfModule>
+<Directory "${SNAP}/htdocs">
+    # Include ownCloud's .htaccess file directly. In a typical setup this would
+    # be dangerous since it increases the capability of the .htaccess file in
+    # case an attacker was able to modify it, but that's not actually possible
+    # on Snappy (since the .htaccess file is read-only) so we'll do it here so
+    # as to avoid manually copying it in and needing to maintain it.
+    Include ${SNAP}/htdocs/.htaccess
```

Comment:
> Unused? We store the config in the config folder on the writeable partition.

Indeed, but that folder isn't reachable by URL (only the in-app `config/` folder is, which is unused).

> There is one in data. It's the same as the one found in config.

Yeah, same situation. Both of those are only useful if Apache directly serves those directories, which it doesn't since both are outside `htdocs/`. In fact, the only way I'd be able to include those `.htaccess` files is if I had `Directory` directives for them. Might as well just leave them un-served, I say.